pkg/tekton/metapipeline/clientfactory.go

```diff
@@ -41,6 +42,12 @@ type clientFactory struct {
 	kubeClient   kubernetes.Interface
 	ns           string
 
+	git    gits.Gitter
+	stderr io.Writer
```

Comment:
so `stderr` is not a terminal.FileWriter?

Reply:
Nope.

Reply:
if it were me, I'd have a wrapper which wraps all three streams as their underlying io.[Reader|Writer] with a function to wrap then as a terminal xyz.

Reply:
Yeah, I'm gonna open an issue for general refactoring around the streams - starting with the struct idea you had.